lm_tournament_eval/scripts/script_args.py

```diff
@@ -139,96 +209,16 @@ def run_tournament():
 
         args.model_args = args.model_args + ",trust_remote_code=True"
 
-    if args.db_path is not None:
-        db = ScoreDatabase(args.db_path)
-
-    logging.info(f"Selected Tasks: {task_names}")
+    return args
 
+def setup_filter_list(args, task_names):
     if ',' in args.filter:
         filter_list = args.filter.split(',')
         if len(filter_list) != len(task_names):
             raise ValueError(
-                f"Filter list length {len(filter_list)} does not match task list length {len(task_list)}. Provide one filter per task."
+                f"Filter list length {len(filter_list)} does not match task list length {len(task_names)}. Provide one filter per task."
```

Comment:
I'll edit this on the `group_fix` branch, as we can have a list of filters but only one task (ie the root task for the group)